admin_ui_support/modules/admin_ui_widget_example/webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
```

Comment:
This file should probably made a script which is reusable.

Reply:
A node module other projects can depend on?

Reply:
It feels like best would be to serve it as part of the admin-ui package people install anyway probably?

Reply:
I think this is done.